.github/workflows/cypress-third-party.yml

```diff
@@ -0,0 +1,139 @@
+name: CI - Cypress (e2e) 3rd party donation tests
```

Comment:
Could this be made DRYer?  Maybe a composite action could be used to hold all the config that's common to this and `cypress.yml`.

@nhcarrigan do you know if composite actions would help, here?

Reply:
Nevermind, we can try to improve this after we've got it working on main.